core/src/main/java/org/jdbi/v3/core/collector/EnumSetCollectorFactory.java

```diff
@@ -39,10 +39,14 @@ public boolean accepts(Type containerType) {
         return findGenericParameter(containerType, EnumSet.class);
     }
 
-    @SuppressWarnings("unchecked")
     @Override
     public Collector<?, ?, ?> build(Type containerType) {
-        Class<? extends Enum> componentType = (Class<? extends Enum>) findGenericParameter(containerType, EnumSet.class)
+        return build0(containerType);
```

Comment:
Could you give it a more functional name and/or comment why it exists?

Reply:
I don't mind the name, but a comment about ecj would be good